Source/Core/Common/IOFile.cpp

```diff
@@ -62,12 +63,20 @@ void IOFile::Swap(IOFile& other) noexcept
   std::swap(m_good, other.m_good);
 }
 
-bool IOFile::Open(const std::string& filename, const char openmode[])
+bool IOFile::Open(const std::string& filename, const char openmode[], SharedAccess sh)
```

Comment:
Should this perhaps log anything out on non-windows systems that the passed in shared access will be ignored?

Maybe `sh` should be tagged as `[[maybe_unused]]` as well

Reply:
> Should this perhaps log anything out on non-windows systems that the passed in shared access will be ignored?

It won't be necessary since it's the default behaviour on the other systems _(i.e. Default == Read)_. I can already access the PCAP file from Linux and Android builds since `fopen`/`fdopen` don't lock the file. Unlike Windows' [fopen_s](https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/fopen-s-wfopen-s?view=msvc-170#remarks) that doesn't allow share read.

> Maybe `sh` should be tagged as `[[maybe_unused]]` as well

Done.